server/auth/testing_utils.go

```diff
@@ -0,0 +1,37 @@
+package auth
```

Comment:
I think we can merge this.
I will repair the test when I'm resolving the #28 .
So since I will be modifying the interface it is better to write test in that ticket.